app/src/main/java/com/lastaoutdoor/lasta/ui/screen/login/components/LoginContent.kt

```diff
@@ -73,3 +82,144 @@ fun LoginContent(onLoginClick: () -> Unit) {
             })
       }
 }
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun LoginPager() {
+
+  val pagerState = rememberPagerState(pageCount = { 4 })
+
+  Column(
+      modifier = Modifier.fillMaxHeight(0.65f),
+      verticalArrangement = Arrangement.SpaceEvenly,
+      horizontalAlignment = Alignment.CenterHorizontally) {
+
+        // The pager composable is a horizontal pager that allows users to swipe through different
+        // pages of content
+        HorizontalPager(
+            state = pagerState,
+            modifier = Modifier.testTag("HorizontalPager").fillMaxHeight(0.9f)) { page ->
+              Column(
+                  modifier = Modifier.fillMaxWidth().fillMaxHeight(),
+                  horizontalAlignment = Alignment.CenterHorizontally,
+                  verticalArrangement = Arrangement.SpaceEvenly) {
+                    PagerContent(page = page)
+                  }
+            }
+        HorizontalPagerIndicator(
+            pageCount = pagerState.pageCount,
+            currentPage = pagerState.currentPage,
+            targetPage = pagerState.targetPage,
+            currentPageOffsetFraction = pagerState.currentPageOffsetFraction)
+      }
+}
+
+@Composable
+private fun HorizontalPagerIndicator(
```

Comment:
There's a lot of computations in this function that don't really see fit in a `@Composable` function. Maybe you can separate them in small helper functions to increase readability/understanding ?

Reply:
Creating helper functions for padding calcultions didn't seem something natural at all for our project, therefore I simplified the orginal code.

It is now way easier to read and has less useless math that achieved only little to the global look.